source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting Started with Solidus Starter Frontend
+date: 2021-11-25
+tags: frontend
+author: George Mendoza
+cover_image: /blog/2021/11/25/getting-started-with-solidus-starter-frontend/home.png
```

Comment:
@kennyadsl I may need some help to make this cover image nicer:

![image](https://user-images.githubusercontent.com/61476/144563696-89b77ee9-0410-46b0-8298-f4a96ecd5ef2.png)

Reply:
@mfrecchiami ping 🙂

Reply:
@gsmendoza added to my ToDo list, I'll tackle it tomorrow!